configs/mozilla-central-mappings.js

```diff
@@ -9,25 +9,26 @@ const EXCLUDED_FILES = {
   "devtools-connection": "devtools/shared/flags",
   "chrome-remote-interface": "devtools/shared/flags",
   "devtools-launchpad": "devtools/shared/flags",
+  "devtools-services": "Services"
```

Comment:
Why is this moved here? 

I tried to make the name and comment explicit enough so that only files mapped to "devtools/shared/flags" should be in this object.

Reply:
nice catch - yeah, i think i was pretty careless here...